test/hostname-policy.json

```diff
@@ -1,4 +1,8 @@
 {
+  "ExactBlacklist": [
```

Comment:
I don't think it does. We could leave it out of this PR. I just wanted a complete example file to load.